cxx-sensors/src/main/java/org/sonar/cxx/sensors/pclint/CxxPCLintSensor.java

```diff
@@ -130,15 +135,22 @@ private boolean isInputValid(@Nullable String file, @Nullable String line, @Null
        * Concatenate M with the MISRA rule number to get the new rule id to save
        * the violation to.
        */
-      private String mapMisraRulesToUniqueSonarRules(String msg) {
+      private String mapMisraRulesToUniqueSonarRules(String msg, char ruleType) {
```

Comment:
I had the idea to have different messages for each MISRA standards in the function, but unless I need only two message type you are right, using boolean is better.